lib/errors.js

```diff
@@ -111,21 +130,29 @@ function JSONAPIErrorHandler (err, req, res, next) {
  * @param {String} errorDetail error message for the user, human readable
  * @param {String} errorCode internal system error code
  * @param {String} errorName error title for the user, human readable
- * @param {String} propertyName for validation errors, name of property validation refers to
+ * @param {String} errorSource Some informations about the source of the issue
```

Comment:
information (singular)

Reply:
didn't know about that, alway happy to learn :)